doc/multiple_robot_arms/multiple_robot_arms_tutorial.rst

```diff
@@ -0,0 +1,442 @@
+        <xacro:gazebo-joint joint="$(arg arm_id_2)_joint5" transmission="hardware_interface/PositionJointInterface" />
+        <xacro:gazebo-joint joint="$(arg arm_id_2)_joint6" transmission="hardware_interface/PositionJointInterface" />
+        <xacro:gazebo-joint joint="$(arg arm_id_2)_joint7" transmission="hardware_interface/PositionJointInterface" />
+
+        <!-- right hand joints control interface -->
+        <xacro:gazebo-joint joint="$(arg arm_id_1)_finger_joint1" transmission="hardware_interface/EffortJointInterface" />
+        <xacro:gazebo-joint joint="$(arg arm_id_1)_finger_joint2" transmission="hardware_interface/EffortJointInterface" />
+
+        <!-- left hand joints control interface -->
+        <xacro:gazebo-joint joint="$(arg arm_id_2)_finger_joint1" transmission="hardware_interface/EffortJointInterface" />
+        <xacro:gazebo-joint joint="$(arg arm_id_2)_finger_joint2" transmission="hardware_interface/EffortJointInterface" />
+
+        <!-- load ros_control plugin -->
+        <gazebo>
+            <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+                <robotNamespace>/panda_multiple_arms</robotNamespace>
+            </plugin>
+        </gazebo>
+
+    </robot>>
+
+Notes: 
+
+1. The ``franka_description`` package is already installed as a dependency of the ``panda_moveit_config`` package. When modeling your robot, make sure the robot_descriptionb package is available in your ROS workspace.
+
+2. We usually need to have a careful look at the arm's xacro file to understand the xacro parameters to use. Here is an example from the ``panda_arm.xacro`` in the ``franka_description`` package: ::
+      
+        <xacro:macro name="panda_arm" params="arm_id:='panda' description_pkg:='franka_description' connected_to:='' xyz:='0 0 0' rpy:='0 0 0' gazebo:=false safety_distance:=0">
+
+
+We can search those parameters in the xacro macro file to understand the function of each. The ``arm_id`` sets a prefix to the arm name to be enable reusing the same model. This is essential for our purpose of modeling multiple arms or robots. The ``connected_to`` parameter gives possibility to connect the robot base with a fixed joint to a given link. In our multiple arms model, each robot is connected to the box shaped base. The gazebo parameter decides whether to load the gazebo simulation required information (e.g links inertia and joints transmission) or not. 
+
+At this point, it is recommended to check our xacro model is working as expected. This can be done in three simple steps; convert your xacro model to URDF, check the connections between links and joints are correct, and if needed you can visualize it (as described before). Run the following commands to check the URDF has no problems. 
+
+    cd ~ws_moveit
+    catkin build 
+    source devel/setup.bash
+    roscd dual_panda_arms/robot_description
+    rosrun xacro xacro panda_multiple_arms.xacro -o panda_multiple_arms.urdf
```

Comment:
I'm getting the following error here:
```
robot_description$ rosrun xacro xacro panda_multiple_arms.xacro -o panda_multiple_arms.urdf
[Errno 2] No such file or directory: 'inertial.yaml' 
when evaluating expression 'xacro.load_yaml('inertial.yaml')'
when instantiating macro: inertial_props (/home/ubuntu/ws_moveit/src/franka_ros/franka_description/robots/utils.xacro)
instantiated from: link_with_sc (/home/ubuntu/ws_moveit/src/franka_ros/franka_description/robots/utils.xacro)
instantiated from: panda_arm (/home/ubuntu/ws_moveit/src/franka_ros/franka_description/robots/panda_arm.xacro)
in file: panda_multiple_arms.xacro
```
Is there a workaround for this?

Reply:
This was a problem in the franka_description package, and I fixed it [here](https://github.com/frankaemika/franka_ros/pull/261). I will include a link to the correct franka package in the tutorial, and improve the content according to your comments. Thanks for the review @jackcenter !